package.json

```diff
@@ -75,7 +75,8 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "rimraf": "^5.0.0",
-    "stylelint": "^15.0.0"
+    "stylelint": "^16.0.0",
+    "stylelint-scss": "^6.0.0"
```

Comment:
Is this needed if it's already installed in `styleling-config-carbon`'s package.json?